proc_IR.py

```diff
@@ -63,7 +56,7 @@
         ph0 = ph0/abs(ph0)
     s /= ph0
     fl.next('frequency domain -- after hermitian function test and phasing')
-    #s.ft('t2')
+    s.ft('t2')
```

Comment:
the title of the plot is "frequency domain", so it doesn't make sese to not FT here -- also, I think it likely would mess up things further down in the code...